TLS 1.3 narrows the time for replay attacks (#110)
huitema/dnsoquic: draft-ietf-dprive-dnsoquic.md

This risk is in fact a subset of the general problem of observing the behavior
of the recursive resolver discussed in "DNS Privacy Considerations"
{{?RFC9076}}. The attack is partially mitigated by reducing the observability

> of this traffic. The risk is also largely mitigated by the replay protection 
> of TLS 1.3 {{?RFC8446}}, which is mandatory to implement.

Reviewer: ```suggestion
of this traffic. The mandatory replay protection mechanisms in
TLS 1.3 {{?RFC8446}} limit but do not eliminate the risk of replay.
```

Author: Adopted the suggestion.
